timesketch/tsctl.py

```diff
@@ -49,14 +49,18 @@ def cli():
 
 
 @cli.command(name="list-users")
-def list_users():
+@click.option("--status", "-s", is_flag=True, help="Show status of the users.")
+def list_users(status):
     """List all users."""
     for user in User.query.all():
         if user.admin:
             extra = " (admin)"
         else:
             extra = ""
```

Comment:
It is not related to the PR, but what do you think about moving the admin marker inside the status option?

Reply:
Fair point. But I think not to change something because something external might break would hinder any improvement in general so should not be a factor.
However, for this specific suggestion, there is no technical need to change it, so let's keep it as is.